tests/unit/computeapi/cuda/transform_compute.cu

```diff
@@ -25,17 +25,24 @@ typedef hpx::compute::cuda::default_executor executor_type;
 typedef hpx::compute::cuda::allocator<int> target_allocator;
 typedef hpx::compute::vector<int, target_allocator> target_vector;
 
+struct transform_test
+{
+    template< typename T >
+    HPX_HOST_DEVICE int operator()(T A, T B)
```

Comment:
Please don't use capital letters for something which is not-a-macro and not-a-template-parameter. Also, should the arguments be `const&`?